app/templates/sign-up.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="row padded-centered-row">
+  <h2 class="bold-h2">Sign up</h2>
+  <form {{on "submit" this.createUser}}>
+    <div class="form-field">
+      <Input @class="form-control" @id="email" @name="email" @placeholder="name@example.com" @type="email" @value={{this.email}} />
```

Comment:
The Input component is a helper provided by Ember that sets up tracking for the value. So all I have to do in the controller is mark email as @tracked.